lib/tasks/vmodl_helper.rb

```diff
@@ -232,6 +236,6 @@ def wsdl_constantize(type)
     type = 'ManagedObject' if type == 'ManagedObjectReference'
 
     type = type.camelcase
-    type.safe_constantize || "RbVmomi::BasicTypes::#{type}".safe_constantize || "RbVmomi::VIM::#{type}".safe_constantize
+    type.safe_constantize || "RbVmomi::BasicTypes::#{type}".safe_constantize || "#{wsdl_to_rbvmomi_namespace(@wsdl)}::#{type}".safe_constantize
```

Comment:
When adding new types in another service class (e.g. `RbVmomi::PBM`) this was failing because it couldn't constantize in `BasicTypes` or `VIM`.  Rather than add `|| PBM.safe_constantize || SMS.safe_constantize || etc...` we can find the `RbVmomi` namespace automatically from the `@wsdl.targetnamespace`